### ISSUE TITLE
Movie playback (on chapters 1-8? only) bypasses saved setting in windows volume mixer

### ISSUE THREAD

**Issue by drojf** [07th-mod/higurashi-assembly#119]
**Reproduction:** 

- If you use Windows Volume Mixer to adjust the game's volume while the game is running, everything works correctly.
- If you restart the game, the Opening Video will play at full volume, even when windows volume mixer shows 10% volume
- If you then move the windows volume mixer slider slightly, the opening video will play at the correct volume

I think maybe because it might spawn the video player in a new process, butthen when you go to adjust it windows realizes which process it is attached to and refreshes the volume.

----

I'd probably class this issue as wontfix, because you can get around it by adjusting the 3 in-game volume sliders, and that works correctly (the BGM Volume slider adjusts the opening video volume). 

In other words, "don't use windows volume mixer" is the solution. Unless there is an easy way to fix this for ch1-8

----

Since Chapter 9 and hou+ use unity's built in video player, I'm assuming they don't have this issue, but I have not checked.
